src/app/shared/utils/utils.ts

```diff
@@ -119,6 +119,20 @@ export class Utils {
         ret = this.numberFormat(ret, decimals);
         return ret + " " + unit;
     }
+    /***
+     * returns the capacity value in appropriate display units
+     */
+    static formatBytes(bytes, decimals = 2) {
+        if (bytes === 0) return '0 Bytes';
+    
+        const k = 1024;
```

Comment:
Can this const be kept in another constant file? Also, it need to be renamed as GiBConverter=1024 and GBConverter=1000 in constant file

Reply:
Will make this change.

Reply:
Is this change done?